tests/rust/log_benchmark/src/image.rs

```diff
@@ -1,12 +1,11 @@
 #![allow(clippy::unwrap_used)]
 
-// About 1gb of image data.
-const IMAGE_DIMENSION: u64 = 16_384;
+const IMAGE_DIMENSION: u64 = 1024;
 const IMAGE_CHANNELS: u64 = 4;
 
 // How many times we log the image.
 // Each time with a single pixel changed.
-const NUM_LOG_CALLS: usize = 4;
+const NUM_LOG_CALLS: usize = 20_000;
```

Comment:
this benchmark exists also for C++ and Python and was meant to compare those. If you change this one, you should also change the others!